### ISSUE TITLE
[Feature]: <Add more validations for sequlize constraints>

### ISSUE THREAD

**Issue by Bhanu-code** [Bhanu-code/db-drawer#9]
### Description

Currently the project has validations for primary key, unique, allowNull etc constrainsts. add more validations for like cascading etc for sequelize.

### Suggested Solution

Checkout the methods that a sequelize instance provide. uncomment the log statements in sqlz.js and try understanding the logs

**Comment by Mohini17o3** [comment]
@Bhanu-code Hi , I'm Mohini , a Gssoc'24 contributor , I would like to work on this issue , please assign this to me .

**Comment by Bhanu-code** [comment]
@Mohini17o3 any upadates?

**Comment by Mohini17o3** [comment]
Hi ,I tried running the visualiser for sequelize model separately but it throws an error ,  still for now I tried adding validations for- on delete , on update  and custom validations , please check if I'm heading in the right direction  ,
![Screenshot (553)](https://github.com/Bhanu-code/db-drawer/assets/143266565/a9c8ce23-9833-415f-8692-0225f93d0bf7)
![Screenshot (555)](https://github.com/Bhanu-code/db-drawer/assets/143266565/89ab4d28-b694-4349-b38d-24c9a451f69e)

**Comment by Bhanu-code** [comment]
@Mohini17o3 have you watched the demonstration video of the project?

**Comment by Mohini17o3** [comment]
> @Mohini17o3 have you watched the demonstration video of the project?

yes , I had , I get this error on running it for sequelize project :  
![Screenshot (556)](https://github.com/Bhanu-code/db-drawer/assets/143266565/159739f5-b670-4026-8b12-1c9f5b297323)
